### PR TITLE
[node-analyzer]add namespace field [admission-controller] add option to set priorityClassName

### PR DESCRIPTION
**Pull request by a1ex-var1amov**
## What this PR does / why we need it:

The PR includes:

- an option to define `namespace` for node-analyzer by default
- an option to set the `priorityClassName` for AC pods (webhook and scanner)

Tested with `ct` (aka `chart-testing`)

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped for the respective charts
- [x] Changelog updated for the charts with version updates.
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Review by yashgiri — APPROVED** [review]
LGTM.

File: charts/node-analyzer/templates/_helpers.tpl [hunk 1]
```diff
@@ -269,3 +269,10 @@ Deploy on GKE autopilot
         true
     {{- end -}}
 {{- end -}}
+
+{{/*
+Returns the namespace for installing components
+*/}}
+{{- define "nodeAnalyzer.namespace" -}}
+    {{- coalesce .Values.namespace .Values.global.clusterConfig.namespace .Release.Namespace -}}
+{{- end -}}
```
Comment (ironashram) [inline charts/node-analyzer/templates/_helpers.tpl]: please add a newline

**Review by ironashram — CHANGES_REQUESTED** [review]
the AC chart uses `chart-doc-gen` to generate the `README`
please run `make docs` and commit the generated changes, otherwise when i trigger the CI workflow it will fail

**Review by ironashram — APPROVED** [review]
lgtm now, the PR title could be improved but that's not a blocker for now, it will be after https://github.com/sysdiglabs/charts/pull/754

File: ct.yaml [hunk 2]
```diff
@@ -1,3 +0,0 @@
-remote: origin
-chart-dirs:
-    - charts
```
Comment (aroberts87) [inline ct.yaml:3]: Just curious, why was this removed?

Comment (a1ex-var1amov) [reply ct.yaml:3]: I think the ct util does it automatically; should we put it back?

**Review by regiluze — APPROVED** [review]
